src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1008.PhysicalLocationPropertiesMustBeConsistent_Invalid.sarif

```diff
@@ -28,6 +28,154 @@
               }
             }
           ]
+        },
+        {
+          "ruleId": "TST0002",
+          "level": "error",
+          "message": {
+            "text": "Bad result - partially overlapping region and contextregion (line/column based)."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "src/test.c"
+                },
+                "region": {
+                  "startLine": 1,
+                  "startColumn": 5,
+                  "endLine": 10,
+                  "endColumn": 5
+                },
+                "contextRegion": {
+                  "startLine": 1,
+                  "startColumn": 1,
+                  "endLine": 10,
+                  "endColumn":15
```

Comment:
>15 [](start = 30, length = 2)

This looks like it fully contains `region` -- it starts before `region` (column ` instead of column 5) and ends after it (column 15 instead of column 5). This looks like a bug in the overlap logic.

Reply:
oops, u r right, let me cross check the code.

---
In reply to: [443867468](https://github.com/microsoft/sarif-sdk/pull/1925#discussion_r443867468) [](ancestors = 443867468)